app/src/main/java/sdp/moneyrun/ui/game/GameLobbyActivity.java

```diff
@@ -247,17 +250,17 @@ public void onCancelled(@NonNull DatabaseError error) {
     @Override
     protected void onDestroy() {
         super.onDestroy();
```

Comment:
This method is really hard to understand, maybe add comments and simplifly it. For example, add block
``